File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let Currentcount=0;
    let Maxcount=0;
    for(let i=0;i<nums.length;i++){
        if(nums[i]===1){
            Currentcount++;
         }
        else{
        Maxcount=Math.max(Currentcount,Maxcount);
        Currentcount=0;
        }
        
    }
    
    return Math.max(Currentcount,Maxcount);
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int currCount=0;
        int maxCount=0;
        for(int i: nums){
            if(i==1){
                currCount++;
            }else{
                maxCount=Math.max(maxCount,currCount);
                currCount=0;
            }
        }
        return Math.max(maxCount,currCount);
    }
}